velox/docs/functions/spark/datetime.rst

```diff
@@ -66,10 +66,24 @@ These functions support TIMESTAMP and DATE input types.
         SELECT dayofweek('2009-07-30'); -- 5
         SELECT dayofweek('2023-08-22 11:23:00.100'); -- 3
 
-.. function:: dow(x) -> integer
+.. spark::function:: dow(x) -> integer
 
     This is an alias for :func:`day_of_week`.
 
+.. spark::function::from_unixtime(unixTime, format) -> string
+
+    Adjusts ``unixTime``(elapsed seconds since UNIX epoch) to configured session timezone, then
```

Comment:
nit: It would be nice to clarify the type of unixTime. I believe it is BIGINT. Does that match Spark?

Reply:
@mbasmanova, yes, unixTime is BIGINT type, matched with Spark. Just clarified in the code.